usaspending_api/bulk_download/v2/views.py

```diff
@@ -255,19 +255,23 @@ def verify_requested_columns_available(sources, requested):
 
 class BulkDownloadListAgenciesViewSet(APIView):
     modified_agencies_list = os.path.join(settings.BASE_DIR,
-                                          'usaspending_api', 'data', 'agency_list_broker_s3.csv')
+                                          'usaspending_api', 'data', 'modified_authoritative_agency_list.csv')
```

Comment:
If the point of bringing back the `modified_authoritative_agency_list` is to include the FREC and IS_FREC columns, could we just use `authoritative_agency_list` so no agencies are left out?

Reply:
I'm bringing back the `modified_authoritative_agency_list` until we come to a consensus on an alternative.